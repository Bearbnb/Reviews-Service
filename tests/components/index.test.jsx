import { shallow, mount, render } from 'enzyme';
import toJson from 'enzyme-to-json';
import React from 'react';
import App from '../../client/components/index';

describe('Index Component', ()=>{
	it('Index should render App to the DOM', () => {
	  const wrapper = shallow(<App />);
	  expect(toJson(wrapper)).toMatchSnapshot();
	});
})
