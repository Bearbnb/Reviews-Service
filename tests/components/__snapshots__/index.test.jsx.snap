// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index Component Index should render App to the DOM 1`] = `
<input
  onClick={[Function]}
  type="submit"
/>
`;
